requirements_dev.txt

```diff
@@ -0,0 +1,20 @@
+#
+# This file is autogenerated by pip-compile
+# To update, run:
+#
+#    pip-compile requirements_dev.in
+#
+appdirs==1.4.4            # via black
+black==20.8b1             # via -r requirements_dev.in
+certifi==2020.6.20        # via requests
+chardet==3.0.4            # via requests
+click==7.1.2              # via black
+idna==2.10                # via requests
+mypy-extensions==0.4.3    # via black
+pathspec==0.8.0           # via black
+regex==2020.9.27          # via black
+requests==2.24.0          # via -r requirements.in
+toml==0.10.1              # via black
+typed-ast==1.4.1          # via black
+typing-extensions==3.7.4.3  # via black
+urllib3==1.25.10          # via requests
```

Comment:
Silly question: these comments were autogenerated, right?

Reply:
All comment's within the requirements.txt files are autogenerated when running the corresponding `pip-compile requirements.in` command. As more transitive dependencies are added and more direct dependencies depending on a specific version, the direct dependency will also be added to the comment.